<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btEnviar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbMensaje">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txtMensaje">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mensaje:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lMensajes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
